index_originl.html

```diff
@@ -55,6 +55,15 @@ <h3 class="suggestion">Suggestions</h3>
 
     </form>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+    Hello Camper!
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+    - The freeCodeCamp Team
+    -->
```

Comment:
Please, avoid leaving comments on your code when submitting
😎😎

Reply:
- [ ] Please make sure your review includes an explanation, a "why", for the requested change. This will help make the student understand the reason for the changes requested. :pray:  Here you are saying that the student should ` avoid leaving comments on your code when submitting` which is not enough to confuse the student to understand your suggestion. :blush: